src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -147,10 +132,15 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <!-- If running from a package these values need to be set again with the resolved IlcPath -->
     <ItemGroup>
-      <PrivateSdkAssemblies Include="$(IlcPath)\sdk\*.dll" />
+      <PrivateSdkAssemblies Include="$(IlcPath)\sdk\*.dll">
+        <IsTrimmable>true</IsTrimmable>
+      </PrivateSdkAssemblies>
     </ItemGroup>
     <ItemGroup>
-      <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" />
+      <!-- Exclude System.IO.Compression.Native.dll for now https://github.com/dotnet/corert/issues/5496 -->
```

Comment:
We have the `FrameworkAssemblies` definition duplicated for reasons I don't remember anymore. This occurrence didn't have the workaround and it kills the compiler with `TrimMode=copy`